src/Nethermind/Nethermind.DataMarketplace.Consumers/Deposits/Domain/DepositDetails.cs

```diff
@@ -1,16 +1,16 @@
 //  Copyright (c) 2018 Demerzel Solutions Limited
```

Comment:
I guess your IDE automatically edited these files?

Reply:
Yes. It's clang formatter plus white spaces removal plugins in vs community edition